lib/Dialect/FIRRTL/Translation/FIRParser.cpp

```diff
@@ -4,9 +4,9 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "circt/FIRParser.h"
 #include "FIRLexer.h"
 #include "circt/Dialect/FIRRTL/Ops.h"
+#include "circt/Dialect/FIRRTL/Translation/Import.h"
```

Comment:
I believe the [standard](https://llvm.org/docs/CodingStandards.html#include-style) is to keep the include for the "main module header" first. Also, should the .cpp file be renamed to `Import.cpp` to match the header?

I think this comment applies to the other translations as well.

Reply:
Right.  Please keep the header and implementation in sync, but in this case, I think FIRParser.h is better than Import.h

Reply:
Thanks for the feedback! I'll move it back to `FIRParser.h`